azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/middleware/CallingMiddleware.kt

```diff
@@ -81,6 +81,9 @@ internal class CallingMiddlewareImpl(
                 is CallingAction.CallStartRequested -> {
                     callingMiddlewareActionHandler.startCall(store)
                 }
+                is CallingAction.CallRequestedWithoutSetup -> {
+                    callingMiddlewareActionHandler.onCallScreenLaunch(store)
```

Comment:
callingMiddlewareActionHandler.onCallScreenLaunch is misleading.
Cold you please rename it to something like CallRequestedWithoutSetup

Reply:
Good call. I checked it. I don't see memory leak. The reason being, it happens when call evicted or network fails, this comes as sdk event so the call sdk resource hold up is already cleaned. 
With no skip setup -> we used to navigate to setup screen 
With setup screen -> we would just navigate to exit.